src/fu-release.c

```diff
@@ -940,6 +940,9 @@ fu_release_load(FuRelease *self,
 					   self);
 	}
 
+	/* don't exercise requirement checks for get-details */
+	return TRUE;
+
```

Comment:
don't you need some sort of check here?

Reply:
Yes it should, though the load-releases is share code between install tasks and get-details / get-updates it seems harder to make a check here. Ideas is appreciated.

Reply:
I think the idea was we'd do something like:

	request = fu_engine_request_new(FU_ENGINE_REQUEST_KIND_ONLY_SUPPORTED);
	fu_engine_request_set_feature_flags(request, ~0);

but that's not happening for get-details. Will ponder.